package.json

```diff
@@ -5,12 +5,11 @@
   },
   "devDependencies": {
     "browser-sync": "^2.8.0",
-    "browserify": "^10.2.6",
-    "browserify-shim": "^3.8.9",
     "cf-component-demo": "git://github.com/cfpb/cf-component-demo.git#0.8.2",
     "chai": "^2.3.0",
+    "dateformat": "^1.0.11",
     "del": "^1.2.0",
-    "fs": "0.0.2",
+    "fs": "latest",
```

Comment:
I realized we don't need `fs`, it's built into node. I mean what is [this](https://www.npmjs.com/package/fs)???

Fixed in https://github.com/cfpb/cfgov-refresh/commit/f116853001dad01f81ff76c22fe0129f31badb49